Modules/posixmodule.c

```diff
@@ -5309,52 +5311,68 @@ os_spawnve_impl(PyObject *module, int mode, path_t *path, PyObject *argv,
 
 
 #ifdef HAVE_FORK
+
+/* Helper function to validate arguments.
+   Returns 0 on success.  1 on failure with a TypeError raised.
```

Comment:
it doesn't really matter.  0 on success and non-zero on failure is the real idiom.  1 and -1 are both non-zero and pass a truth test.